daprdocs/content/en/js-sdk-docs/_index.md

```diff
@@ -0,0 +1,193 @@
+---
+type: docs
+title: "Dapr JavaScript SDK"
+linkTitle: "JavaScript"
+weight: 1000
+description: JavaScript SDK packages for developing Dapr applications
+---
+
+## Pre-requisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [Node version 15 or greater](https://nodejs.org/en/) 
+
+## Installing and importing Dapr's JS SDK
+
+
+Install the SDK with npm:
+```
+npm i @roadwork/dapr-js-sdk
+```
+
+Import the libraries for the the given protocol you're using:  
+
+```javascript
+import { DaprClient, DaprServer } from "@roadwork/dapr-js-sdk/http";
```

Comment:
@XavierGeerinck  Thanks, So just to clarify, the  server will take in all 4 params including daprHost and daprPort? I ask because i'm looking at your actors branch reference and it only take in serverHost and serverPort: https://github.com/dapr/js-sdk/blob/actors/examples/grpc/hello-world/src/index.ts#L15

Possible you're in the process of updating now.